src/VerifyExamples/Ast/Intermediate.elm

```diff
@@ -29,7 +28,16 @@ type Prefix
 
 fromString : String -> List IntermediateAst
 fromString =
-    commentLines >> List.filterMap toIntermediateAst
+    commentLines >> List.map atLeastASpace >> List.filterMap toIntermediateAst
+
+
+atLeastASpace : String -> String
+atLeastASpace str =
```

Comment:
☁️ A comment explaining why this is needed would be useful.

Reply:
totally. 😊